en/docs/reference/connectors/mongodb-connector/mongodb-connector-overview.md

```diff
@@ -13,10 +13,13 @@ It is always recommended to download the latest version of the connector.
 ## Compatibility
 
 | Connector version | Supported product versions |
-| ----------------- | ----------------------------- |
-| 1.0.0             | APIM 4.0.0, EI 7.1.0, EI 6.6.0    |
+|-------------------|----------------------------|
+| 2.0.0             | MI 4.3.0                   |
```

Comment:
Should we add 4.1.0 as well?

Reply:
We might have to add the update level for 4.3.0.

Reply:
Updated the 4.3.0 with the update level. 

Should we add 4.1.0? Since this is the MI 4.3.0 doc I thought we don't need to add it here. Please let me know if you think it should be added here.